spec/turn_spec.rb

```diff
@@ -0,0 +1,53 @@
+require './lib/turn'
+require './lib/card'
+
+RSpec.describe Turn do
+  it 'exists' do
+    card = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    turn = Turn.new("Juneau", card)
+
+
+    expect(turn).to be_instance_of(Turn)
+    expect(card).to be_instance_of(Card)
+  end
+
+  it 'has a guess' do
+    card = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    turn = Turn.new("Juneau", card)
+
+    expect(turn.guess).to eq("Juneau")
+  end
+
+  it 'has a card' do
+    card = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    turn = Turn.new("Juneau", card)
+
+    expect(turn.card.question).to eq("What is the capital of Alaska?")
+    expect(turn.card.answer).to eq("Juneau")
+    expect(turn.card.category).to eq(:Geography)
+  end
```

Comment:
for this spec, we mainly only need to test that turn.card returns the card object you initialized the object with. We don't need to check the attributes on that card object because we've already tested that in the Card class.